app/api/resources/rapid_view.py

```diff
@@ -48,27 +68,69 @@ async def resolve_species(
             query_params = parse_qs(query_string, separator=";")
 
             url = construct_url(genome_id, subpath, query_params)
-            return resolved_response(url, request)
+            response = RapidResolverResponse(
+                response_type=RapidResolverHtmlResponseType.INFO,
+                code=308,
+                resolved_url=url,
```

Comment:
The [ticket](https://embl.atlassian.net/browse/ENSWBSITES-2952) also mentions multiple responses (code 300), which is not addressed here. In principle an accession ID can resolve to multiple genome UUIDs (if it's part of both partial and integrated release, like the human reference) but I shouldn't apply to rapid species so it's fine (assuming any rapid assembly won't become a part of a partial release in the future).